proto/acl-config.proto

```diff
@@ -13,5 +13,10 @@ message ACLConfig {
   // adminSKIEncoded - ACL admin public key SKI in hex format
   string adminSKIEncoded = 2 [(validate.rules).string.pattern = "^[0-9a-f]+$"];
   // validators - set of validators public keys
-  repeated string validators = 3;
+  repeated ACLValidator validators = 3;
+}
+
+message ACLValidator {
+  string publicKey = 1;
+  string keyType = 2;
```

Comment:
Maybe in the proto-file explicitly list the types of supported keys and enum instead of a string 

enum KeyType {
    ED... = 0;
    GOST... = 1;
    EDCA.... = 2;
  }

  KeyType type = 1;